src/main/ui/app/route/TagPage.jsx

```diff
@@ -0,0 +1,93 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+
+import {Component} from "react";
+import fetch from "isomorphic-fetch";
+import AddOnList from "../component/AddOnList";
+
+export default class TagPage extends Component {
```

Comment:
I didn't add it to the existing search page only due to one reason. Even there is a perfect tag match, the search results display more modules than the modules that actually have the tag.
It may be noted that the highest rank results are always the ones with the matching tag but it still returns more.
Example : https://addons-stg.openmrs.org/#/search?&q=form-entry 
Actual modules with the tag are only the Top 4 but it returns more than 10 results